src/main.c

```diff
@@ -85,7 +85,7 @@ void EnableVCountIntrAtLine150(void);
 
 void AgbMain()
 {
-    RegisterRamReset(RESET_ALL);
+    RegisterRamReset(MODERN ? RESET_ALL & ~RESET_IWRAM : RESET_ALL);
```

Comment:
You might want to comment this switch. Could be pretty confusing if you don't know why you have to do it.

Reply:
`//Exclude IWRAM from resetting if compiled with modern gcc`?

Reply:
Observe the "outdated" tag please

Reply:
Oh yeah, sorry, didn't know that tag was a thing. Thanks